recipes/breakpad/all/conanfile.py

```diff
@@ -0,0 +1,118 @@
+    topics = ["crash", "report", "breakpad"]
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://chromium.googlesource.com/breakpad/breakpad/"
+    settings = 'os', 'compiler', 'build_type', 'arch'
+    options = {
+        "fPIC": [True, False]
+    }
+    default_options = {
+        "fPIC": True
+    }
+    _env_build = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def requirements(self):
+        if self.settings.os == "Linux":
+            self.requires("linux-syscall-support/cci.20200813")
+
+    def _patch_sources(self):
+        # Use Conan's lss instead of the submodule
+        # 1. Remove from include dirs
+        # 2. Remove from list of headers to install
+        # 3. Patch all #include statements
+        tools.replace_in_file(os.path.join(self._source_subfolder, "Makefile.in"),
+                            "$(includegbc_HEADERS) $(includelss_HEADERS) ",
+                            "$(includegbc_HEADERS) "
+        )
+        tools.replace_in_file(os.path.join(self._source_subfolder, "Makefile.in"),
+                            "install-includelssHEADERS install-includepHEADERS ",
+                            "iinstall-includepHEADERS "
+        )
+        files_to_patch = [
+            "src/tools/linux/md2core/minidump-2-core.cc",
+            "src/processor/testdata/linux_test_app.cc",
+            "src/common/memory_allocator.h",
+            "src/common/linux/memory_mapped_file.cc",
+            "src/common/linux/file_id.cc",
+            "src/common/linux/safe_readlink.cc",
+            "src/client/minidump_file_writer.cc",
+            "src/client/linux/handler/exception_handler.cc",
+            "src/client/linux/handler/exception_handler_unittest.cc",
+            "src/client/linux/log/log.cc",
+            "src/client/linux/crash_generation/crash_generation_client.cc",
+            "src/client/linux/minidump_writer/linux_dumper.cc",
+            "src/client/linux/minidump_writer/linux_dumper_unittest_helper.cc",
+            "src/client/linux/minidump_writer/proc_cpuinfo_reader.h",
+            "src/client/linux/minidump_writer/minidump_writer.cc",
+            "src/client/linux/minidump_writer/linux_ptrace_dumper.cc",
+            "src/client/linux/minidump_writer/cpu_set.h",
+            "src/client/linux/minidump_writer/directory_reader.h",
+            "src/client/linux/minidump_writer/line_reader.h"
+        ]
+
+        for file in files_to_patch:
+            tools.replace_in_file(
+                os.path.join(self._source_subfolder, file),
+                "#include \"third_party/lss/linux_syscall_support.h\"",
+                "#include <linux_syscall_support.h>"
+            )
+
+        # Let Conan handle fPIC
```

Comment:
Is this something we want? Or should we assume that if it is hardcoded is "for a reason" and don't mess with it?

Reply:
This is hardcoded so users don't have to change anything to use it in a shared library.